packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v3.0.1 (Sep 03, 2024)
+- Fix Cookie `expires` time [#1994](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1994)
```

Comment:
```suggestion
- Fixed an issue where the `expires` flag of the cookie was mistakenly set using days instead of the correct unit, seconds. [#1994](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1994)
```